snakemake/__init__.py

```diff
@@ -1623,6 +1660,47 @@ def get_argument_parser(profile=None):
         " between S3 bucket and the run environment (container) is automatically"
         " handled by Tibanna.",
     )
+    group_google_life_science.add_argument(
+        "--google-life-sciences",
```

Comment:
@moschetti is there an official short name? --google-lsa?

Reply:
Haha, whenever I read that I read "Google Lisa" and my mind wanders off to the Simpsons :) Lisa would definitely use Snakemake!

Reply:
No, there's not an official short name. 

For reference, Nextflow used `google-pipelines` as the old API name was Pipelines API. So I wouldn't use this.

Maybe go with --google-lifesciences (as one word to match the API namespace though).

Reply:
okay I am updating all references (variables and otherwise) of "google_life_sciences" to "google_lifesciences" so we can be consistent.